app/jobs/publish_feed_job.rb

```diff
@@ -159,4 +161,19 @@ def mismatched_publishing_item?(podcast, pub_item)
 
     mismatch
   end
+
+  private
+
+  def set_job_id_on_publishing_item(podcast)
+    current_pub_item = PublishingQueueItem.current_unfinished_item(podcast)
+
+    return unless current_pub_item && current_pub_item.job_id.nil?
+
+    current_pub_item.update!(job_id: job_id)
+    Rails.logger.info("Set job_id on PublishingQueueItem", {
```

Comment:
When would this log statement be used for?  Seems like I'd only go looking for it if a PublishingQueueItem was missing it, right?